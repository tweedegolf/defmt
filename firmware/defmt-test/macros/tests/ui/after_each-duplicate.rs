fn main() {}

#[defmt_test_macros::tests]
mod tests {
    #[after_each]
    fn first() {}

    #[after_each]
    fn second() {}
}
