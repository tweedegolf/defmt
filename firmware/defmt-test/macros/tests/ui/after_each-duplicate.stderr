error: only a single `#[after_each]` function can be defined
 --> tests/ui/after_each-duplicate.rs:9:8
  |
9 |     fn second() {}
  |        ^^^^^^
